deltachat-ios/Handler/AccountSwitcherHandler.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AccountSwitcherHandler.swift
+//  deltachat-ios
+//
+//  Created by bb on 30.09.22.
+//  Copyright © 2022 Jonas Reinsch. All rights reserved.
```

Comment:
that should be removed, too :)

Reply:
really wondering where this comes from, is there some old template deep in xcode?

Reply:
removed with the latest commit.